crypto/evp/e_chacha20_poly1305.c

```diff
@@ -290,12 +290,12 @@ static int chacha20_poly1305_cipher(EVP_CIPHER_CTX *ctx, unsigned char *out,
 
             Poly1305_Update(POLY1305_ctx(actx), temp, POLY1305_BLOCK_SIZE);
         }
-        Poly1305_Final(POLY1305_ctx(actx), ctx->encrypt ? actx->tag
+        Poly1305_Final(POLY1305_ctx(actx), EVP_CIPHER_CTX_encrypting(ctx) ? actx->tag
```

Comment:
This line goes past 80 columns, needs being broken up differently.  I suggest breaking the line after the comma.